### DISCUSSION TITLE
TO MAINNET TEA

### DISCUSSION THREAD

**Discussion by VeroTestnet** [teaxyz/white-paper#144] [Q&A]
What are you proud of and what do you hope for if tea opens mainnet and is successful out there?

**Comment by nellybtc** [comment]
I love the what they are building

**Comment by ronliang521** [comment]
I like the concept of Tea. I have been following them for a long time and have been waiting for the testnet to open today. I feel very accomplished, just like watching my own child grow up. I will continue to pay attention to and support this tea in the future.
